computer_science_course_prereqs.csv

```diff
@@ -0,0 +1,41 @@
+Course,Prerequisite(s),,
+CS 1308,,,
+CS 1319,,,
+CS 1428,"MATH 1315, MATH 1317, MATH 1319, MATH 1329, MATH 2417, MATH 2471",,
```

Comment:
this is a question
should we simply list explicit pre-reqs?

for example `Math 1319` is a prereq.
 `MATH 1317` is a prereq of  `MATH 1319`

so we do we need to list both?

this one is a suggestion.

I think it would be easier to simply have multiple rows for
`CS 1428` each line lists one prerequisite.

Reply:
so all of the comma seperated prereqs in the same column are options for prereqs to the same class. i don't feel that we can simply ignore them

Reply:
any ideas how to address this?